lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,22 @@
+<%page expression_filter="h" args="notification_type, notification_icon, notification_message, hint_notification"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "info"
+    notification_icon = "question"
+    notification_message = "Hint (1 of 2): A fruit is the fertilized ovary from a flower. Hint (2 of 2): A fruit contains seeds of the plant."
+    hint_notification = True
+%>
+
+<div class="notification ${notification_type}">
+    <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span>
```

Comment:
OK, happy to touch base.  I just saw that the icon took a variable but I didn't see any textual representation of the icon's value.  A good way to test this is to imagine what would happen if the image and CSS didn't load (simulating the experience of the non-visual user).  Would you be able to figure out what kind of message it was BEFORE reading the message, or would you have to read the message and then come to the conclusion that it was an Error, or a Hint, or a Success message.